VoxelSniperCore/src/main/java/com/github/kevindagame/brush/polymorphic/PolyBrush.kt

```diff
@@ -165,8 +174,12 @@ class PolyBrush(
                 )
             }
         }
+        if (params.size == 1) {
+            v.owner().player.sendMessage(Messages.PARAMETER_PARSE_ERROR.replace("parameter", params[0]))
+            return
+        }
         for (property in properties) {
-            if (params[0].equals(property.name, ignoreCase = true)) {
+            if (property.name.equals(params[0]) || property.aliases.any { it.equals(params[0], ignoreCase = true) }) {
```

Comment:
Maybe better to also ignore case for the name itself for consistency.
```suggestion
            if (property.name.equals(params[0], ignoreCase = true) || property.aliases.any { it.equals(params[0], ignoreCase = true) }) {
```